main/uart_nic.c

```diff
@@ -46,7 +46,11 @@
 int ieee80211_output_pbuf(esp_aio_t *aio);
 esp_err_t mac_init(void);
 
-#define FW_VERSION 11
+#define FW_VERSION 12
+
+#define SCAN_MAX_STORED_SSIDS 64
```

Comment:
Is this based on anything, or just „randomly chosen number“? Considering we likely have enough RAM on the ESP side, it could make sense to just overestimate by a bit (I can imagine someone seeing 64 or more SSIDs in eg. a big city somewhere).

Reply:
Since the terrible results in the office, I don't think the small esp with crappy antenna will ever find more than few AP around. Of course I can increase it if needed. I can put 128, but would love to test it more if it doesn't break anything.